### DISCUSSION TITLE
1.20 Forge

### DISCUSSION THREAD

**Discussion by Wh1rr** [HttpRafa/modflared#23] [Ideas]
*Not an idea, only one discussion category exists*

Since the forge folder was re-named to neoforged, I assume forge is no longer supported within the new 1.20 releases. Just wanted to check

**Comment by HttpRafa** [comment]
Because many big mods switched to NeoForge, I thought that it was no longer worth supporting Forge. But if there is a lot of demand I would have no problem supporting it again.

**Reply by Wh1rr** [reply]
Ah yeah, was looking at modpacks rather than individual mods. But yes, newer packs will be on neoF so shouldn't be necessary, thanks 😄
